Formula/openshift-cli.rb

```diff
@@ -37,15 +34,7 @@ def install
     # See https://github.com/golang/go/issues/26487
     ENV.O0 if OS.linux?
 
-    args = ["cross-build-#{os}-#{arch}"]
-    args << if build.stable?
-      "WHAT=cmd/oc"
-    else
-      "WHAT=staging/src/github.com/openshift/oc/cmd/oc"
-    end
-    args << "SHELL=/bin/bash" if OS.linux?
-
-    system "make", *args
+    system "make", "cross-build-#{os}-#{arch}", "OS_GIT_VERSION=#{version}", "SHELL=/bin/bash"
```

Comment:
Simplify build as `/bin/bash` is also available on macOS and I don't think `WHAT` is used anymore.

I did try `make oc` locally as it would allow removing OS/arch, but it failed on `gss_const_name_t`.

---

`OS_GIT_VERSION` should help get a more useful `oc version` number (not sure if this or `SOURCE_GIT_TAG` is better to set).

- Existing bottle:
    ```
    Client Version: v4.2.0-alpha.0-1570-g3c85519
    Kustomize Version: v4.5.4
    ```
- New stable build:
    ```
    Client Version: 4.12.0-202208031327
    Kustomize Version: v4.5.4
    ```
- Head build:
    ```
    Client Version: HEAD-5b29f86
    Kustomize Version: v4.5.4
    ```
- Official Pre-built (doesn't seem like 4.12.0 actually released yet? only found EC builds)
    ```
    Client Version: 4.12.0-ec.2
    Kustomize Version: v4.5.4
    ```